doc/model_persistence.rst

```diff
@@ -1,294 +1,307 @@
+GitHub. Persisted models are often deployed in production using containers
+like Docker, in order to freeze the environment and dependencies.
+
+If you want to know more about these issues please refer to these talks:
+
+- `Adrin Jalali: Let's exploit pickle, and skops to the rescue! | PyData
+  Amsterdam 2023 <https://www.youtube.com/watch?v=9w_H5OSTO9A>`__.
+- `Alex Gaynor: Pickles are for Delis, not Software - PyCon 2014
+  <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`__.
+
+
+.. _serving_environment:
+
+Replicating the training environment in production
+..................................................
+
+If the versions of the dependencies used may differ from training to
+production, it may result in unexpected behaviour and errors while using the
+trained model. To prevent such situations it is recommended to use the same
+dependencies and versions in both the training and production environment.
+These transitive dependencies can be pinned with the help of `pip`, `mamba`,
+`conda`, `poetry`, `conda-lock`, `pixi`, etc.
+
+|details-start|
+**InconsistentVersionWarning**
+|details-split|
+
+When an estimator is loaded with a scikit-learn version that is inconsistent
+with the version the estimator was pickled with, a
+:class:`~sklearn.exceptions.InconsistentVersionWarning` is raised. This warning
+can be caught to obtain the original version the estimator was pickled with::
+
+  from sklearn.exceptions import InconsistentVersionWarning
+  warnings.simplefilter("error", InconsistentVersionWarning)
+
+  try:
+      est = pickle.loads("model_from_prevision_version.pickle")
+  except InconsistentVersionWarning as w:
+      print(w.original_sklearn_version)
+
+|details-end|
+
+
+Serving the model artifact
+..........................
+
+The last step after training a scikit-learn model is serving the model.
+Once the trained model is successfully loaded it can be served to manage
+different prediction requests. This can involve deploying the model as a
+web service using containerization, or other model deployment strategies,
+according to the specifications. In the next sections, we will explore
+different approaches to persist a trained scikit-learn model.
+
+
+Summarizing the key points
+--------------------------
+
+Based on the different approaches for model persistence, the key points for
+each approach can be summarized as follows:
+
+* `ONNX`: It provides a uniform format for persisting any machine learning
+  or deep learning model (other than scikit-learn) and is useful
+  for model inference. It can however, result in compatibility issues with
+  different frameworks.
+* :mod:`skops.io`: Trained scikit-learn models can be easily shared and put
+  into production using :mod:`skops.io`. It is more secure compared `pickle`
+  based alternate approaches as it does not load arbitrary code unless
+  explicitly asked by the user.
+* :mod:`joblib`: Efficient storage and memory mapping techniques make it faster
```

Comment:
```suggestion
* :mod:`joblib`: Efficient storage and memory mapping techniques make it faster at model loading
```
Or add this info further above. The model prediction won't be faster, just the loading of the model in the prediction service.